deployment/terraform/keycloak.go

```diff
@@ -0,0 +1,279 @@
+package terraform
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/assets"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+)
+
+func (t *Terraform) setupKeycloak(extAgent *ssh.ExtAgent) error {
+	keycloakBinPath := "/opt/keycloak/keycloak-" + t.config.ExternalAuthProviderSettings.KeycloakVersion + "/bin"
+
+	mlog.Info("Configuring keycloak", mlog.String("host", t.output.KeycloakServer.PublicIP))
+
+	command := "start-dev"
+	extraArguments := []string{}
+
+	if !t.config.ExternalAuthProviderSettings.DevelopmentMode {
+		command = "start"
+	}
+
+	sshc, err := extAgent.NewClient(t.output.KeycloakServer.PublicIP)
+	if err != nil {
+		return err
+	}
+
+	// Check if the keycloak database exists
+	result, _ := sshc.RunCommand(`sudo -iu postgres psql -l | grep keycloak 2> /dev/null`)
```

Comment:
Why this `2> /dev/null`? Why not simply pipe to `wc -l`? Also, please check the error here.

Reply:
I was using that to avoid errors here, but piping to `wc -l` is way more elegant. :thumbsup: